src/content_sao.h

```diff
@@ -33,10 +33,18 @@ class UnitSAO: public ServerActiveObject
 	virtual ~UnitSAO() = default;
 
 	virtual void setYaw(const float yaw) { m_yaw = yaw; }
+        virtual void setPitch(const float pitch) { m_pitch = pitch; }
+        virtual void setRoll(const float roll) { m_roll = roll; }
```

Comment:
These 2 lnes seem to have an extra space of indent.

Reply:
No problem :) it's not that messy actually, and pretty good for a first contribution.